analysis/analysis.yml

```diff
@@ -1,9 +1,9 @@
 version: 2
 
 models:
-- name: select_from_orders_example
-  description: >
-    This is an example of an analysis model.
-    dbt's notion of models makes it easy for data teams to version control and collaborate on data transformations. 
-    Sometimes though, a certain sql statement doesn't quite fit into the mold of a dbt model. 
-    These more "analytical" sql files can be versioned inside of your dbt project using the analysis functionality of dbt.
\ No newline at end of file
+  - name: select_from_orders_example
+    description: >
+      This is an example of an analysis model.
+      dbt's notion of models makes it easy for data teams to version control and collaborate on data transformations. 
+      Sometimes though, a certain sql statement doesn't quite fit into the mold of a dbt model. 
+      These more "analytical" sql files can be versioned inside of your dbt project using the analysis functionality of dbt.
```

Comment:
In my VSCode I use the `Insert Final Newline` setting. 

https://thoughtbot.com/blog/no-newline-at-end-of-file#:~:text=So%2C%20it%20turns%20out%20that,is%20a%20line%20%E2%80%9Cterminator%E2%80%9D. :-D